lib/email/receiver.rb

```diff
@@ -190,6 +191,7 @@ def process_internal
           end
         end
 
+        raise ReplyToDigestError if EmailLog.find_by(email_type: "digest", message_id: @mail.in_reply_to).present?
```

Comment:
```suggestion
        raise ReplyToDigestError if EmailLog.find_by(email_type: "digest", message_id: @mail.in_reply_to).exists?
```

Reply:
@eviltrout I commited this and got `undefined method "exists?" for `EmailLog`. Switching back to `.present?`

Reply:
Ah NVM, I switched the `find_by` to `where` so that `.exists?` should work.

Reply:
Oh my bad it should be `EmailLog.where(email_type: "digest", message_id: @mail.in_reply_to).exists?`